sakisandrac/overlook-final-project
# Feature/load bookings

## Additions to project:
- Login page
- Dashboard page
- Render all user bookings to dashboard (using user 50 as the customer)
- Bookings page
- Date picker to select days to book
- Renders all available rooms on any given day

## How to test features:
- Login with the username: _customer50_ and password: _overlook2021_
- Scroll through the box on the bottom of the dashboard page to see all bookings made from user
- Click the new booking button to go to the search page
- Use the date picker to search any day and click search
- See all available rooms for any given day

## Features that are not finished yet:
- Dashboard puts all current and past bookings into one container- they are not separated yet
- Book now button does not work yet
